docs/guides/getting-started-istio.md

```diff
@@ -0,0 +1,214 @@
+## Running Kong Ingress Controller with Istio
+
+By following this guide, you will:
+* install Istio v1.6.7 and Kong in your cluster,
+* deploy an example Istio-enabled application (_bookinfo_),
+* deploy an `Ingress` customized with a `KongPlugin` for the example application,
+* make several requests to the sample application via Kong and Istio,
+* see the performance metrics of the sample application, provided by Istio.
+
+### Prerequisites
+
+* A Kubernetes v1.15-1.18 cluster which can pull container images from public registries. Examples:
+    * A managed Kubernetes cluster (AWS EKS, Google Cloud GKE, Azure AKS),
+    * Minikube,
+    * `microk8s` with the `dns` addon enabled,
+* `kubectl` with admin access to the cluster.
```

Comment:
```suggestion
For this guide, you will need:

* A Kubernetes v1.15 (or newer) cluster which can pull container images from public registries. For example, you can use:
    * A managed Kubernetes cluster (AWS EKS, Google Cloud GKE, Azure AKS).
    * Minikube.
    * `microk8s` with the `dns` addon enabled.
* `kubectl` with admin access to the cluster.
```
Unbounded the version requirements, as I don't expect we'll see issues where newer K8S versions are no longer compatible. Cross that bridge when we get there if so.